ql/ql/src/queries/performance/LargeJoinOrder.ql

```diff
@@ -130,9 +129,9 @@ int getSize(ComputeRecursive recursive, string predicateName, int iteration, TDe
     kind = TCurrent() and
     i = iteration
   |
-    result = getInLayerEventWithName(recursive, predicateName).getDeltaSize(iteration - 1)
+    result = getInLayerEventWithName(recursive, predicateName).getDeltaSize(i)
     or
-    not exists(getInLayerEventWithName(recursive, predicateName).getDeltaSize(iteration - 1)) and
+    not exists(getInLayerEventWithName(recursive, predicateName).getDeltaSize(i)) and
```

Comment:
This is a small drive-by fix. We were incorrectly using `iteration - 1` here (which was only correct when `kind = TPrevious()`).